{
	"id": "oval:org.opensuse.security:def:202328859",
	"version": "1",
	"class": "vulnerability",
	"metadata": {
		"title": "CVE-2023-28859",
		"affected": {
			"family": "unix",
			"platform": [
				"SUSE Linux Enterprise Module for Package Hub 15 SP4",
				"SUSE Linux Enterprise Module for Package Hub 15 SP5"
			]
		},
		"reference": [
			{
				"ref_id": "Mitre CVE-2023-28859",
				"ref_url": "https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2023-28859",
				"source": "CVE"
			},
			{
				"ref_id": "SUSE CVE-2023-28859",
				"ref_url": "https://www.suse.com/security/cve/CVE-2023-28859",
				"source": "SUSE CVE"
			}
		],
		"description": "\n    redis-py before 4.4.4 and 4.5.x before 4.5.4 leaves a connection open after canceling an async Redis command at an inopportune time, and can send response data to the client of an unrelated request. (This could, for example, happen for a non-pipeline operation.) NOTE: the solutions for CVE-2023-28859 address data leakage across AsyncIO connections in general.\n    ",
		"advisory": {
			"from": "security@suse.de",
			"severity": "Moderate",
			"cve": [
				{
					"text": "CVE-2023-28859 at SUSE",
					"href": "https://www.suse.com/security/cve/CVE-2023-28859/",
					"impact": "moderate",
					"cvss3": "4.3/CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:L/I:N/A:N"
				},
				{
					"text": "CVE-2023-28859 at NVD",
					"href": "https://nvd.nist.gov/vuln/detail/CVE-2023-28859",
					"impact": "moderate",
					"cvss3": "6.5/CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:H/I:N/A:N"
				}
			],
			"bugzilla": [
				{
					"text": "SUSE bug 1209811",
					"href": "https://bugzilla.suse.com/1209811"
				},
				{
					"text": "SUSE bug 1209812",
					"href": "https://bugzilla.suse.com/1209812"
				}
			],
			"issued": {
				"date": "2023-09-07"
			},
			"updated": {
				"date": "2024-04-18"
			},
			"affected_cpe_list": {
				"cpe": [
					"cpe:/o:suse:packagehub:15:sp4",
					"cpe:/o:suse:packagehub:15:sp5"
				]
			}
		}
	},
	"criteria": {
		"operator": "AND",
		"criterias": [
			{
				"operator": "OR",
				"criterions": [
					{
						"test_ref": "oval:org.opensuse.security:tst:2009668202",
						"comment": "SUSE Linux Enterprise Module for Package Hub 15 SP4 is installed"
					},
					{
						"test_ref": "oval:org.opensuse.security:tst:2009760609",
						"comment": "SUSE Linux Enterprise Module for Package Hub 15 SP5 is installed"
					}
				]
			}
		],
		"criterions": [
			{
				"test_ref": "oval:org.opensuse.security:tst:2009754974",
				"comment": "python3-redis is not affected"
			}
		]
	}
}
